app/src/createExcelFiles/fetchSearches.js

```diff
@@ -0,0 +1,22 @@
+import { SearchItem } from "./modal";
+
+export async function fetchSearches(searchName, searchNumber) {
```

Comment:
your file name AND method name can use a serious uplift.
What is fetchSearches.js ? why does it have a method fetchSearches as well? it tells me NOTHING about whats happening.

How about a file name like...`GoogleSearch.js` or `GoogleSearchService.js` or `GoogleService.js` or `SearchService.js`.

The method can be... `search(searchQuery, numberOfResults)`

now THAT is a lot more verbose.
If you are going to go with obscure method names and input params, u need to include jsdocs. Otherwise, make ur code readable so that any 5 year old can read n understand whats going on.